src/scene.ts

```diff
@@ -3451,6 +3451,10 @@ export class Scene extends AbstractScene implements IAnimatable {
             this._intermediateRendering = true;
             let needRebind = false;
 
+            for (let step of this._cameraDrawRenderTargetStage) {
+                needRebind = needRebind || step.action(this.activeCamera);

```

Comment:
I needed to move this above the loop below because having a render target like depth in the below loop causes needRebind = true so effectlayer stages don't end up getting run. Is there a reason the _cameraDrawRenderTargetStage steps should not be run if there are render targets?

Reply:
no I guess the || is wrong no need to move it...

It should be a simple | here... my bad idea is it stays true or become true but the action always run...

GG Good catch

Reply:
I'll fix it bu using | instead of ||

Reply:
Here is the fix I ended up using:
```
for (let step of this._cameraDrawRenderTargetStage) {
                needRebind = step.action(this.activeCamera) || needRebind;
            }
```

(Without changing the position)